modules/styling-transform/spec/utils/handleFocusRing.spec.ts

```diff
@@ -7,68 +7,78 @@ describe('handleFocusRing', () => {
     _reset();
   });
 
+  it('should not rewrite focusRing if not called inside a correct static style function', () => {
+    const program = createProgramFromSource(`
+      const styles = styled({ ...focusRing() })
+    `);
+
+    const result = transform(program, 'test.ts', {transformers: [handleFocusRing]});
+
+    expect(result).toContain('styles = styled({ ...focusRing() })');
```

Comment:
is not happy about this

Reply:
looks like this test needs to needs to be updated? https://github.com/Workday/canvas-kit/blob/8651a7204522376431c7d25a7ffe7aecc5c8dac2/modules/styling-transform/spec/utils/handleCreateStyles.spec.ts#L218

Reply:
There's something in the transform that's causing the parent to be `undefined` and failing the lookup test